packages/flutter/test/material/radio_test.dart

```diff
@@ -180,12 +180,11 @@ void main() {
     expect(semantics, hasSemantics(TestSemantics.root(
       children: <TestSemantics>[
         TestSemantics.rootChild(
-          id: 1,
+          id: 2,
           flags: <SemanticsFlag>[
-            SemanticsFlag.isInMutuallyExclusiveGroup,
             SemanticsFlag.hasCheckedState,
             SemanticsFlag.hasEnabledState,
-            SemanticsFlag.isFocusable,
```

Comment:
Uh, we were marking a disabled radio button as focusable before? It is correct, that a disabled element is not focusable, right?